src/config/socket/socket.js

```diff
@@ -45,6 +45,43 @@ export default (io) => {
       }
     });
 
+    socket.on('inviteFriend', ({ username, friendEmail, email }) => {
+      User.update({ email: friendEmail },
```

Comment:
can we have all
`User.update({})` lines in a function, and call the function here